lib/node_modules/@stdlib/lapack/base/iladlr/README.md

```diff
@@ -0,0 +1,232 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# iladlr
+
+> Return last non-zero row of matrix `A`.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var iladlr = require( '@stdlib/lapack/base/iladlr' );
+```
+
+#### iladlr( order, M, N, A, LDA )
+
+Returns last non-zero row of matrix `A`.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var out;
+var A = new Float64Array( [ 1.0, 2.0, 3.0, 4.0 ] );
+
+out = iladlr( 'row-major', 2, 2, A, 2 );
+// returns 1
+```
+
+The function has the following parameters:
+
+-   **order**: storage layout.
+-   **M**: number of rows in `A`.
+-   **N**: number of columns in `A`.
+-   **A**: input [`Float64Array`][mdn-float64array].
+-   **LDA**: stride of the first dimension of `A` (a.k.a., leading dimension of the matrix `A`).
+
+Note that indexing is relative to the first index. To introduce an offset, use [`typed array`][mdn-typed-array] views.
+
+<!-- eslint-disable stdlib/capitalized-comments -->
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+// Initial an array.
```

Comment:
```suggestion
// Initial an array:
```

Nowhere do we end line comments with only a period. Needs to be fixed here and elsewhere.